### PR TITLE
AISDK-198: Add sentiment analysis client to java SDK

### PR DESCRIPTION
**Pull request by kbridbur**

### CODE REVIEW

File: examples/SentimentAnalysis.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+import ai.rev.sentimentanalysis.SentimentAnalysisClient;
+
+import java.io.IOException;
+import java.util.Arrays;
+
+public class SentimentAnalysis {
+
+  public static void main(String[] args) {
+    // Assign your access token to a String
+    String accessToken = "your_access_token";
```
Comment (amikofalvy) [inline examples/SentimentAnalysis.java]: Use our conventional placeholder format

```suggestion
    String accessToken = "<YOUR_ACCESS_TOKEN>";
```

File: src/main/java/ai/rev/sentimentanalysis/SentimentAnalysisClient.java [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+    if (accessToken == null || accessToken.isEmpty()) {
+      throw new IllegalArgumentException("Access token must be provided");
+    }
+    this.client = ClientHelper.createOkHttpClient(accessToken);
+    Retrofit retrofit = ClientHelper.createRetrofitInstance(client, "sentiment_analysis", "v1");
+    this.apiInterface = retrofit.create(SentimentAnalysisInterface.class);
+  }
+  
+  /** Manually closes the connection when the code is running in a JVM */
+  public void closeConnection() {
```
Comment (amikofalvy) [inline src/main/java/ai/rev/sentimentanalysis/SentimentAnalysisClient.java:49]: Is this necessary?

File: src/main/java/ai/rev/sentimentanalysis/models/SentimentAnalysisJob.java [hunk 3]
```diff
@@ -0,0 +1,307 @@
+package ai.rev.sentimentanalysis.models;
+
+import ai.rev.speechtotext.models.asynchronous.RevAiJobType;
+import ai.rev.speechtotext.models.asynchronous.RevAiFailureType;
+import java.util.List;
+
+import com.google.gson.annotations.SerializedName;
```
Comment (amikofalvy) [inline src/main/java/ai/rev/sentimentanalysis/models/SentimentAnalysisJob.java:7]: I saw recently that Slack swapped out gson for Moshi. We might consider the same in the future.

**Review by amikofalvy — APPROVED** [review]

File: src/main/java/ai/rev/sentimentanalysis/SentimentAnalysisClient.java (hunk 2 above)
Comment (kbridbur) [reply src/main/java/ai/rev/sentimentanalysis/SentimentAnalysisClient.java:49]: I believe so. @Chanc3 may have some more info on why this is needed

**Pull request merged**